packages/permissions/src/hooks/use-is-authorized/use-is-authorized.spec.tsx

```diff
@@ -436,11 +469,10 @@ describe('when only one permission matches', () => {
           expect(queryByText('Is authorized: Yes')).toBeInTheDocument();
         });
       });
-
-      describe('has demanded dataFence to VIEW or MANAGE orders on the specific store', () => {
-        it('should indicate as authorized', () => {
+      describe('when dataFence permission is not met', () => {
```

Comment:
I don't know. I prefer the language of `when general permissions...`, `when datafence permissions` vs `when demanded datafence permissions ...`

Reply:
I mean there is a difference between the demanded and the actual and we need to make that difference clear in our test cases.

Reply:
the problem is that it does not make sense to talk about general permissions being view or manage anymore, the dataFence permissions are agnostic to what the general permission are, so in the context of data fences we talk only about wether the user meets the general permissions or not. 

overall, I think the test cases and their structure is good enough for now and would like to get this PR through.